pkg/collector/corechecks/cluster/orchestrator/orchestrator_processor.go

```diff
@@ -240,13 +241,38 @@ func chunkServices(services []*model.Service, chunkCount, chunkSize int) [][]*mo
 	return chunks
 }
 
-// processNodesList process a nodes list into process messages.
-func processNodesList(nodesList []*corev1.Node, groupID int32, cfg *config.OrchestratorConfig, clusterID string) ([]model.MessageBody, error) {
+// processNodesList process a nodes list into nodes process messages and cluster process message.
+func processNodesList(nodesList []*corev1.Node, groupID int32, cfg *config.OrchestratorConfig, clusterID string, client *apiserver.APIClient) ([]model.MessageBody, model.MessageBody, error) {
 	start := time.Now()
 	nodeMsgs := make([]*model.Node, 0, len(nodesList))
+	nodeCount := int32(0)
+	kubeletVersions := map[string]int32{}
+	podCap := uint32(0)
+	podAllocatable := uint32(0)
+	memoryAllocatable := uint64(0)
+	memoryCap := uint64(0)
+	cpuAllocatable := uint64(0)
+	cpuCap := uint64(0)
+
+	apiServerVersions := map[string]int32{}
+	apiVersion, err := client.Cl.Discovery().ServerVersion()
+	if err != nil {
+		log.Errorf("Error getting server apiVersion: %s", err.Error())
+		return nil, nil, err
+	}
+	apiServerVersions[apiVersion.String()] = 1
 
 	for s := 0; s < len(nodesList); s++ {
 		node := nodesList[s]
+		nodeCount += 1
+		kubeletVersions[node.Status.NodeInfo.KubeletVersion] += 1
+		podCap += uint32(node.Status.Capacity.Pods().Value())
+		podAllocatable += uint32(node.Status.Allocatable.Pods().Value())
+		memoryAllocatable += uint64(node.Status.Allocatable.Memory().Value())
+		memoryCap += uint64(node.Status.Capacity.Memory().Value())
+		cpuAllocatable += uint64(node.Status.Allocatable.Cpu().MilliValue())
```

Comment:
MilliValue for cpu is millicore
Value for Pods are pods
Value for Mem are bytes

Reply:
taken from this PR: https://github.com/DataDog/datadog-agent/pull/6437